### ISSUE TITLE
Is `IEEE` format available?

### ISSUE THREAD

**Issue by ntjess** [brechtm/rinohtype#280]
I saw a few commits from around 7c0a8795ea49402ecb407550b51b18977be0fa88 that deal with an IEEE stylesheet. However, I can't seem to find examples from the current repo state. Is this still available? My goal was to create a "Hello world" example that looks like this:

```latex
\documentclass[conference]{IEEEtran}
\usepackage{lipsum}

\title{My Title}
\author{My Name}

\begin{document}
\maketitle
Hello, world!

\lipsum[5]
\end{document}
```

**Comment by brechtm** [comment]
No, that IEEE style sheet was removed back in 2016. See the [full history of ieee.py](/brechtm/rinohtype/commits/HEAD/rinohlib/stylesheets/ieee.py).

With the [last version of `ieee.py`](https://github.com/brechtm/rinohtype/blob/f8f768fd15a8561208a7a6951e678285e03dbf40/rinohlib/stylesheets/ieee.py) and [`sphinx.rts`](/brechtm/rinohtype/blob/master/src/rinoh/data/stylesheets/sphinx.rts) as reference, it should be fairly straightforward to create a new `ieee.rts`. Is that something you'd like to take a shot at?
